app/grandchallenge/reader_studies/models.py

```diff
@@ -1341,6 +1353,11 @@ def assign_permissions(self):
             self.question.reader_study.editors_group,
             self,
         )
+        assign_perm(
+            f"delete_{self._meta.model_name}",
+            self.question.reader_study.editors_group,
```

Comment:
Where is the management command to assign this permission to existing objects?

Reply:
I didn't include it, as you set it could easily be done via the shell:

> Not super necessary, it's a quick thing we can do in the shell:
> 
> ```python
> for rs in ReaderStudy.objects.all():
>   assign_perm("delete_answer", rs.editors_group, rs.answers.all())
> ```

But I can add it if you'd prefer that :slightly_smiling_face: